buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -407,15 +408,19 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
             def TEST_FLAG = (name.contains('+')) ? name.substring(name.indexOf('+')+1).toUpperCase() : ''
             def TEST_JOB_NAME = get_test_job_name(get_target_name(name), SPEC, SDK_VERSION, BUILD_IDENTIFIER)
 
+            def IS_PARALLEL = false
+            if (TEST_JOB_NAME.contains("special.system")){
+                IS_PARALLEL = true
```

Comment:
I'm not sure I like enabling parallel this way. It could work temporarily but I think long term we need a better solution.
Some questions would be
1. Who controls parallel (build/test)
1. Is parallel enabled based on test target or platform or both
1. Is parallel different based on Jenkins server

Perhaps a "smart" system where we determine available machines in order to decide whether to enable parallel.